PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/OriginalGraphTest.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.pl2016gr2.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class OriginalGraphTest extends GraphTest {
+  
+  private OriginalGraph curGra;
+
+  @Override
+  public GraphInterface getInstance() {
+    OriginalGraph gra = new OriginalGraph();
```

Comment:
Local variable <code>gra</code> hides the public <code>gra</code> field of the parent class.